lightspeed-catalog/index.yaml

```diff
@@ -57,7 +57,7 @@ properties:
           }
         ]
       capabilities: Basic Install
-      createdAt: "2024-04-28T22:18:32Z"
+      createdAt: "2024-05-03T12:33:20Z"
```

Comment:
@vbelouso I would have expected this file to be updated too. If its not I have to check the update-bundle-catalog make target.

Reply:
@xrajesh, I have tested it, and this file will update only if I update the bundle image. 
For example, from the tag `0.0.1` -> `0.0.2`